etna/lib/etna/clients/metis/models.rb

```diff
@@ -50,7 +50,10 @@ def to_h
         end
       end
 
-      class ListFolderRequest < Struct.new(:project_name, :bucket_name, :folder_path, keyword_init: true)
+      class FolderRequest < Struct.new(:project_name, :bucket_name, :folder_path, keyword_init: true)
```

Comment:
Just acknowledges that many FolderRequest types have the same fields and an explicit super class makes it a little bit safer to share a request object instantiated once and used in separate operations.